README.md

```diff
@@ -1,12 +1,20 @@
 # Business Site
 
-Replace this readme with your own information about your project. 
+This week we built a business site with a little touch of JavaScript. Focus on a Hero Image or Video, a Signup form, Accordion and the over all requirements:
+- Style your page
+- Site should have a header with responsive image/video
+- Site should have a signup form with at least three different input types
+- Site should have an FAQ accordion, built with JavaScript
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+I took advatage of my own company on this task and used a layout, video, font and some text. I knew this would help me focus more on the new learnings of JavaScript. 
+
+For this week I added more comments and links in the code so you easily can find where I got the code, found some inspiration etc. Hopefully helpful for all.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Well first of all: Hello JavaScript, could you BE anymore confusing!? Haha, I have struggled a bit, googled a lot and did a fair amount of time at Stack Overflow scrolling trough questions and answers. I will trust the process and hopefully it all will make sense. I found a lot of inspiration and help from former BootCamp-students, this years bunch of BootCamp-heroes and my Zebra-gang ;)
 
 ## View it live
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+
+https://movmtform.netlify.app/
```

Comment:
I think your site is great and it follows all of the blue requirements! Keep up the good work :)

Reply:
Thank you so much for the review!